masonry/src/properties/spinner_color.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::any::TypeId;
+
+use crate::core::{Property, UpdateCtx};
+use crate::peniko::color::{AlphaColor, Srgb};
+
+/// The color a spinner is painted with.
```

Comment:
Why does this need something custom rather than a color property? What part of the spinner is painted with this?

Reply:
But you could just not have SpinnerColor in that case... so it is a very simple thing.